lib/native-client.js

```diff
@@ -66,6 +66,15 @@ const SYSTEM = 'system';
  */
 const ADMIN = 'admin';
 
+/**
+ *
+ * @param {Map<K, V>} _map
+ * @returns {V}
+ */
+function getFirstFromMap(_map) {
+  return Array.from(_map.values())[0];
```

Comment:
```suggestion
  return _map.values().next().value;
```

If you want to avoid the extra array copy, which is probably not very relevant here, so feel free to ignore this :)

Reply:
Oooh, good one, thanks!